src/metadata.rs

```diff
@@ -109,6 +109,7 @@ pub struct FighterMeta {
     pub hurtbox: ColliderMeta,
     //Will likely need a hashmap(?) of AttackMetas, fighters will have multiple attacks
     pub attack: AttackMeta,
+    pub attachment: Option<FighterSpritesheetMeta>,
```

Comment:
maybe could decide on another way to represent spritesheets of types other than `FighterSpritesheet`